src/Form/index.tsx

```diff
@@ -417,13 +425,15 @@ function Form({
     updateFieldValues(updatedValues);
   }
 
-  function removeRepeatedRow(index: number) {
+  function removeRepeatedRow(element: any) {
```

Comment:
most of the logic in this function is the same as the logic in the addRepeatedRow function. Can we unify?

Reply:
I don't really agree.  They are significantly different and commonizing them would be difficult with no real gain.